src/js/hca-rjsf/config/form.js

```diff
@@ -674,31 +675,54 @@ const formConfig = {
             'view:spouseIncome': {
               'ui:title': 'Spouse income',
               'ui:options': {
-                hideIf: (formData) => formData.maritalStatus !== 'married' // Something else too?
+                hideIf: (formData) => formData.maritalStatus !== 'Married' // Something else too?
```

Comment:
Thought about it, but it should always be title case, no? It wouldn't add much, to do so, but...Yeah, I guess I'll just play it safe.